janus/lib/client/jsx/project-view.jsx

```diff
@@ -147,11 +192,36 @@ const ProjectView = ({project_name}) => {
   let retrieveProject = useCallback(
     () => {
       json_get(`/api/admin/${project_name}/info`)
-        .then(({project}) => setProject(project))
+      .then(({project}) => {
+        setProject(project);
+        setProjectDetails(project);
```

Comment:
Hmm, what's the difference here between these two?

Reply:
Ah, I see, one of them is the project as it is being edited, and the other is the project since the last load, right?
Nit: Name clarification?